src/neighbor_budget_interp_mod.F90

```diff
@@ -388,13 +398,16 @@ SUBROUTINE interpolate_neighbor_budget_vector(IPOPT,grid_in,grid_out, &
     ! - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
     !  COMPUTE NUMBER OF OUTPUT POINTS AND THEIR LATITUDES AND LONGITUDES.
     IRET=0
-    IF(.not. to_station_points) THEN
-       CALL GDSWZD(grid_out, 0,MO,FILL,XPTS,YPTS, &
-            RLON,RLAT,NO,CROT,SROT)
+    if(to_station_points) then
+       CALL GDSWZD(grid_out, 0,MO,FILL,XPTS,YPTS,RLON,RLAT,NO,CROT,SROT)
```

Comment:
Does this line need to be in both parts of the if/else block or can it be pulled out in front?

Reply:
Yes.  That can be pulled of the if block.  I will fix.